pkgs/applications/video/go2tv/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, buildGoModule, fetchFromGitHub, xorg, libglvnd, pkg-config, withGui ? true }:
+
+buildGoModule rec {
+  pname = if withGui then "go2tv" else "go2tv-lite";
+  version = "1.13.0";
+
+  src = fetchFromGitHub {
+    owner = "alexballas";
+    repo = "go2tv";
+    rev = "v${version}";
+    sha256 = "sha256-ZHKfBKOX3/kVR6Nc+jSmLgfmpihc6QMb6NvTFlsBr5E=";
+  };
+
+  vendorSha256 = "sha256-msXfXFWXyZeT6zrRPZkBV7PEyPqYkx+JlpTWUwgFavI=";
+
+  doCheck = false;
+
+  buildInputs = [
```

Comment:
The withGui boolean must be used somewhere.

Reply:
I've improved it somewhat now, does it make more sense?

Reply:
Ah, that wasn't obvious to me. We don't need to add extra code for that but a nice short comment is enough.

Reply:
I could also make buildInputs an empty list for withGui=false perhaps?

(doesn't change the -lite binary and its dependencies though)